src/libraries/System.Diagnostics.TextWriterTraceListener/src/System/Diagnostics/DelimitedListTraceListener.cs

```diff
@@ -243,7 +243,7 @@ private void EscapeMessage(string message, StringBuilder sb)
         {
             int index;
             int lastindex = 0;
-            while ((index = message.IndexOf('"', lastindex)) != -1)
+            while ((index = message.IndexOf('\"', lastindex)) != -1)
```

Comment:
I'm not clear what's happening here.  Why is this change necessary?  Does this even compile?

Reply:
This (and others like it) was just general cleanup. I can revert if desired.

Reply:
Personally I'd prefer if it was reverted; the original seems better to me.

Reply:
No problem. :)